definitions/npm/javascript-natural-sort_v0.7.x/test_javascript-natural-sort_v0.7.x.js

```diff
@@ -0,0 +1,3 @@
+import naturalSort from 'javascript-natural-sort'
+
+naturalSort('abc', 'bfg')
```

Comment:
You'll need to add a failing test, and place `// $ExpectError` above it. Something like this:

```javascript
// $ExpectError
naturalSort([], {});
```